Analytics/Request/BlockingRequestHandler.cs

```diff
@@ -243,7 +249,7 @@ internal BlockingRequestHandler(Client client, TimeSpan timeout)
 							// If status code is greater than 500 and less than 600, it indicates server error
 							// Error code 429 indicates rate limited.
 							// Retry uploading in these cases.
-							Task.Delay(backoff).Wait();
+							await Task.Delay(backoff).ConfigureAwait(false);
```

Comment:
Why did this change?

Reply:
Since the `MakeRequest` method is async using `Wait()` instead of `await` can cause deadlock. I can move it to another PR if needed, just thought it is too small for separate one.

Reply:
I'd like to be to seperate if you don't mind! It makes it easy to independently revert the header change from this one if needed and also link/attribute properly from the changelog.